connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -836,6 +836,8 @@ private void startGame() {
         startGamePacket.setVanillaVersion("*");
         startGamePacket.setAuthoritativeMovementMode(AuthoritativeMovementMode.CLIENT);
         upstream.sendPacket(startGamePacket);
+
+        getWorldCache().setShowCoordinates(connector.getConfig().isShowCoordinates());
```

Comment:
Nitpickiest thing: can you move this to the `connect` function? I think this function looks cleanest when it's dedicated to the StartGamePacket.